vnext/Microsoft.ReactNative/packages.config

```diff
@@ -6,7 +6,7 @@
   <package id="Microsoft.SourceLink.GitHub" version="1.0.0" targetFramework="native" developmentDependency="true" />
   <package id="Microsoft.UI.Xaml" version="2.5.0" targetFramework="native" />
   <package id="Microsoft.Windows.CppWinRT" version="2.0.200615.7" targetFramework="native" />
-  <package id="Microsoft.WinUI" version="3.0.0-preview4.210210.4" targetFramework="native"/>
-  <!-- package id="ReactNative.Hermes.Windows" version="0.7.2" targetFramework="native" / -->
+  <package id="Microsoft.WinUI" version="3.0.0-preview4.210210.4" targetFramework="native" />
```

Comment:
@asklar does this mean we're always pulling in WinUI3 bits?

Reply:
for the M.RN solution yes. For the app itself it's dynamically determined at build time during autolinking.

Reply:
Is it restored for solutions depending on the MSRN project?

Reply:
Confirmed this is pulled in on user solutions. Will file an issue.